pkg/utils/utils.go

```diff
@@ -595,3 +603,12 @@ func generateRandomGuid() net.HardwareAddr {
 
 	return guid
 }
+
+func GetNicSriovMode(pciAddress string) (string, error) {
```

Comment:
@pliurh Will `SetNicSriovMode` be in a follow up?

Reply:
No, at least not for openshift. The plan is to use the Machine Config Operator to insert a new systemd service which will configure the VF and set NIC to switchdev mode on the host.

Reply:
In this release, we only use switchdev for OVN offloading use case. And we want users can use VF bond with it. In openshift-sdn, bond is created by NetworkManager configure file inserted by MCO. So we have to configure switchdev before NetworkManager starts on the host.